src/components/ScaleBuilder.vue

```diff
@@ -154,9 +165,17 @@ const updatebaseMidiNote = debounce((event: Event) => {
         </div>
 
         <div class="control">
+          <button
+            @click="autoFrequency"
+            style="position: absolute; right: 1.4rem; z-index: 9001"
```

Comment:
I know this isn't good CSS. @SeanArchibald you will need to help me here.

Reply:
This is a good solution for now and I might end up changing the position of the button in future anyway which will make things simpler